docs/testing.md

```diff
@@ -88,29 +88,34 @@ create_lb: true
 ```
 
 #### For executing locally via docker
-- Optional flags that can be added in `.env` file. Default values are set on entrypoint/deployrancher/rancher_suite_test.go
+- Optional flags that can be added in `.env` file. Default values are set on `entrypoint/deployrancher/rancher_suite_test.go`
 ```
 CERT_MANAGER_VERSION=v1.13.3
 CHARTS_VERSION=v2.7.12
-CHARTS_REPO_NAME=<helm repo name>
-CHARTS_REPO_URL=<helm chart repo url>
-CHARTS_ARGS=bootstrapPassword=admin,replicas=1 #(Comma separated helm chart args)
-RANCHER_VERSION=v2.7.12
+CHARTS_REPO_NAME=<charts repo name>
+CHARTS_REPO_URL=<charts repo url>
+CHARTS_ARGS=bootstrapPassword=admin,replicas=1 #(Comma separated chart args)
+RANCHER_REPO_NAME=<rancher repo name>
+RANCHER_REPO_URL=<rancher repo url>
```

Comment:
Is it possible to get a drop down of only the repo names. and based on the repo name, we have hard coded urls. 
The reason am asking is, i feel the URLs are pre-determined/fixed at this point, and we dont need the user to know which URL to provide? 
We also need to document, which repo name + url we are using / or user should provide here - all possible options.

Reply:
We want to keep it flexible to be able to use with prime configs as well, added `latest` as default on jenkins job so that user can update as needed